dpe/src/dpe_instance.rs

```diff
@@ -73,12 +68,11 @@ impl DpeInstance<'_> {
         Ok(dpe)
     }
 
-    pub fn new_for_test<'a>(
+    pub fn new_for_test(
```

Comment:
nit: could just remove this. But that would require a large-scale find/replace, so I understand if you don't want to in this PR 😄

Reply:
Done.